query/src/storages/fuse/io/block_reader.rs

```diff
@@ -44,13 +50,15 @@ pub struct BlockReader {
     arrow_schema: Arc<Schema>,
     projected_schema: DataSchemaRef,
     parquet_schema_descriptor: SchemaDescriptor,
+    ctx: Arc<QueryContext>,
 }
 
 impl BlockReader {
     pub fn create(
         operator: Operator,
         schema: DataSchemaRef,
         projection: Vec<usize>,
+        ctx: Arc<QueryContext>,
```

Comment:
Put the ctx as the first parameter of the function?
That's OK to fix it in the later PR :)

Reply:
:ok_hand: